app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1>Help</h1>
+<p>
+Get help on the Ruby on Rails Tutorial at the
+link_to href="http://www.railstutorial.org/help">Rails Tutorial
```

Comment:
wrong syntax
you can refer at https://apidock.com/rails/ActionView/Helpers/UrlHelper/link_to

Reply:
please fix your cmt